packages/web-app-files/src/mixins/actions/fetch.js

```diff
@@ -47,7 +56,7 @@ export default {
         headers
       })
         .then((r) => r.blob())
```

Comment:
can we us async await here?

Reply:
Not sure about the implications, @kulmann ?